sig-and-wg/sig-core/README.md

```diff
@@ -4,7 +4,7 @@ The purpose of this group is to:
 * Publicly represent Kyma
 * Facilitate the collaboration with external contributors
 
-## Responsibilities
+Responsibilities
```

Comment:
Is it required at all?

Reply:
why not remove both "Responsibilities" & "The SIG Core:" and merge the two bullet. points lists under "The purpose of this group is to: " . You might need to adjust the wording of the second list and that's all.

Reply:
Good point